src/button/interrupt.rs

```diff
@@ -59,20 +62,19 @@ pub fn enable(external_interrupts: &EXTI, sysconfig: &SYSCFG, mode: TriggerMode)
 }
 
 fn configure_exti0(interrupt_mask: &IMR1) {
-    interrupt_mask.modify(|_, w| w.mr0().set_bit())
+    interrupt_mask.modify(|_, w| w.mr0().unmasked())
 }
 
 fn map_exti0_to_pa0(external_interrupt_config: &EXTICR1) {
-    const PORT_A_CONFIG: u8 = 0x000;
-    external_interrupt_config.modify(|_, w| unsafe { w.exti0().bits(PORT_A_CONFIG) });
+    external_interrupt_config.modify(|_, w| w.exti0().pa0());
```

Comment:
Heh. I wonder when the HAL added this.
When I first wrote this, I had to use the direct write in order to avoid having a memory safe handle to the PORTA config register. 
I want to look at this more closely.